src/protect-function-app/greeting/__init__.py

```diff
@@ -0,0 +1,47 @@
+"""
+This is an Azure Function that response at GET /api/greeting.

```

Comment:
Can you double check this sentence?

Reply:
Good catch, I'll fix that mess tomorrow.

Reply:
@skabou updated in https://github.com/Azure-Samples/ms-identity-docs-code-python/pull/10/commits/93880be4ceaea459a654acc42cb41584cb319750